daemons/controld/controld_remote_ra.c

```diff
@@ -70,18 +70,29 @@ typedef struct remote_ra_cmd_s {
     uint32_t status;
 } remote_ra_cmd_t;
 
-enum remote_migration_status {
-    expect_takeover = 1,
-    takeover_complete,
+#define remote_set_flags(remote_flags, call_for, flags_to_set) do { \
+    remote_flags = pcmk__set_flags_as(__func__, __LINE__, \
+                                      LOG_TRACE, "remote", (call_for), (remote_flags), \
+                                      (flags_to_set), #flags_to_set); \
+        } while (0)
+
+#define remote_clear_flags(remote_flags, call_for, flags_to_clear) do { \
+    remote_flags = pcmk__clear_flags_as(__func__, __LINE__, \
+                                        LOG_TRACE, "remote", (call_for), (remote_flags), \
+                                        (flags_to_clear), #flags_to_clear); \
+        } while (0)
```

Comment:
Similarly you can pass ra_data instead of ra_data->status (or even lrm_state, which would let you use lrm_state->node_name instead of call_for)